packages/app-config-writer/test/fixtures/navigation-config/ui5-app-no-inbounds/webapp/manifest.json

```diff
@@ -0,0 +1,167 @@
+{
+  "_version": "1.38.0",
+  "sap.app": {
+    "id": "testNameSpace.alpv2",
+    "type": "application",
+    "i18n": "i18n/i18n.properties",
+    "applicationVersion": {
+      "version": "0.0.1"
+    },
```

Comment:
Is it possible to maybe reduce the test manifest files to what is being needed for the configuration?
This would also make it easier to check the snapshots for expected changes.

Reply:
OK will do...just wanted to use a realistic manifest.